crates/nodeapiserver/proto/OpenAPIv2.proto

```
@@ -0,0 +1,665 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// THIS FILE IS AUTOMATICALLY GENERATED.
+
+syntax = "proto3";
+
+package openapi.v2;
```

Buf (1.72.0)

[error] Files with package "openapi.v2" must be within a directory "openapi/v2" relative to root but were in directory "crates/nodeapiserver/proto".

(PACKAGE_DIRECTORY_MATCH)